frontend/src/components/__tests__/CheckData.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import CheckData from "../CheckData";
+
+test("renders a file input element", async () => {
+  const wrapper = render(
+    <CheckData
+      data={[]}
+      advanceStep={() => {}}
+      backStep={() => {}}
+      onCancel={() => {}}
+      setSelectedHeaders={() => {}}
+      setHiddenColumns={() => {}}
+      selectedHeaders={new Set<string>()}
+      hiddenColumns={new Set<string>()}
+      register={() => {}}
+    />
+  );
+  expect(wrapper.container).toMatchSnapshot();
+});
```

Comment:
the title of the unit test is not accurate. Also this snapshot test is not that useful because it renders an empty component without data. If you want to keep this as snapshot test, at least pass some data to the component so that it renders a table in my opinion.

Reply:
Fixed